packages/terra-dropdown-button/translations/en-AU.json

```diff
@@ -1,4 +1,6 @@
 {
   "Terra.dropdownButton.moreOptions": "More Options",
-  "Terra.dropdownButton.selected": "Selected."
+  "Terra.dropdownButton.selected": "Selected.",
+  "Terra.dropdownButton.of": "of",
+  "Terra.dropdownButton.expanded": "Expanded button."
```

Comment:
we need to log a translation Jira for expanded `Terra.dropdownButton.expanded` for other locales

Reply:
Raised translation jira  https://jira.cerner.com/browse/TRANS-14527